contentctl/objects/baseline.py

```diff
@@ -18,23 +15,25 @@
 
 
 class Baseline(SecurityContentObject):
-    # baseline spec
-    #name: str
-    #id: str
-    #version: int
-    #date: str
-    #author: str
-    #contentType: SecurityContentType = SecurityContentType.baselines
+    name:str = Field(...,max_length=67)
     type: Annotated[str,Field(pattern="^Baseline$")] = Field(...)
     datamodel: Optional[List[DataModel]] = None
-    #description: str
     search: str = Field(..., min_length=4)
     how_to_implement: str = Field(..., min_length=4)
     known_false_positives: str = Field(..., min_length=4)
     tags: BaselineTags = Field(...)
 
     # enrichment
     deployment: Deployment = Field({})
+    
+    def get_conf_stanza_name(self, app:CustomApp, max_stanza_length:int=81)->str:
+        stanza_name = f"{app.label} - {self.name}"
```

Comment:
This needs to be redfined because Baseline does not inherit from Detection.
It would be a good idea to make both Baseline and Detection inherit from a common Abstract Parent class, but that is a big change and not included in this PR.

Reply:
LGTM